src/ixdat/tools.py

```diff
@@ -273,3 +277,20 @@ def tstamp_to_yyMdd(tstamp: float) -> str:
         year % 100, chr(ord("A") + month - 1), day
     )
     return date_string
+
+
+def tstamp_to_string(tstamp: float) -> str:
+    """Return a string representation if unix timestamps `tstamp`"""
+    dt = datetime.datetime.fromtimestamp(tstamp)
+    string_format = ixdat.config.config.timestamp_string_format
+    if string_format == "native":
```

Comment:
Actually the only usages PyCharm can find are:

![image](https://github.com/ixdat/ixdat/assets/1611076/a7c0c75b-2906-4b94-a2ff-d9beb4a7152e)

so, it is used in a method in `Measurement` named after the date format itself, so that one will **NEED** to return `yyMdd` no matter what. But what I could do, is to make the format in `tstamp_to_string` overridable via a keyword argument and **then** I would be able to delete the old one and use the new. What do you think about that?

Reply:
Sounds good!

Reply:
I added the keyword argument, replaced the one usages and deprecated the old function (since it is technically public interface and we therefore can't just delete it)